scripts/launch/connect-to-cluster/default.nix

```diff
@@ -18,7 +18,9 @@
 , debug ? false
 , disableClientAuth ? false
 , extraParams ? ""
-, useStackBinaries ? false
+# one of "nix", "stack" or "path"
+, binaryMethod ? "nix"
+, forceDontCheck ? false
```

Comment:
I don't get this change?

Reply:
an unrelated change that got partially polled in by the cherry-pick, i could remove it from this PR